src/plugins/intel_cpu/src/plugin.h

```diff
@@ -49,6 +49,9 @@ class Engine : public InferenceEngine::IInferencePlugin {
 
     void GetPerformanceStreams(Config &config, const std::shared_ptr<ngraph::Function>& ngraphFunc);
 
+    Config MergeConfig(const std::map<std::string, std::string>& config);
```

Comment:
I would say this method introduce more confusion than benefit.
I would propose to remove it at all and leave previous logic untouched inside the code:
```
    Config conf = engConfig;
    conf.readProperties(config);
```

Reply:
ok, updated